### PR TITLE
Create journal-of-mountain-science.csl

### PR DESCRIPTION
**Pull request by nilsnerd**
Difference to Springer Author Date Style: the bibliography is ordered by appearance in the text in the case of same first author same year, ignoring second and all other authors, ignoring title.

### CODE REVIEW

File: journal-of-mountain-science.csl [hunk 1]
```diff
@@ -0,0 +1,179 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" default-locale="en-US" version="1.0" demote-non-dropping-particle="never">
+  <info>
+    <title>Journal of Mountain Science</title>
+    <title-short>JMS</title-short>
+    <id>http://www.zotero.org/styles/journal-of-mountain-science</id>
+    <link href="http://www.zotero.org/styles/journal-of-mountain-science" rel="self"/>
+    <link rel="template" href="https://www.zotero.org/styles/springer-basic-author-date-no-et-al"/>
```
Comment (adam3smith) [inline journal-of-mountain-science.csl]: this needs to be http not https for legacy reasons

File: journal-of-mountain-science.csl [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+    <title-short>JMS</title-short>
+    <id>http://www.zotero.org/styles/journal-of-mountain-science</id>
+    <link href="http://www.zotero.org/styles/journal-of-mountain-science" rel="self"/>
+    <link rel="template" href="http://www.zotero.org/styles/springer-basic-author-date-no-et-al"/>
+    <link href="http://www.springer.com/cda/content/document/cda_downloaddocument/Guide+to+authors-Journal+of+Mountain+Science-20160412.docx?SGWID=0-0-45-1557769-p173618726" rel="documentation"/>
+    <link href="http://jms.imde.ac.cn/c/document_library/get_file?uuid=8b0ad56b-a83e-440b-885c-c0e4228984a1&amp;groupId=21915" rel="documentation"/>
+    <link href="http://jms.imde.ac.cn/c/document_library/get_file?uuid=d0228dde-7349-4513-a8d2-db36cb352e46&amp;groupId=21915" rel="documentation"/>
+    <link href="http://jms.imde.ac.cn/web/21915/guide-to-author" rel="documentation"/>
+    <link href="http://jms.imde.ac.cn/web/21915/download-center" rel="documentation"/>
+    <link href="http://www.springer.com/earth+sciences+and+geography/journal/11629" rel="documentation"/>
```
Comment (rmzelle) [inline journal-of-mountain-science.csl]: @nilsnerd, do we need 6 "documentation" links? Maybe you could reduce it to the most relevant 1-2?

File: journal-of-mountain-science.csl [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+    <author>
+      <name>Niels Schwab</name>
+      <email>niels.schwab@uni-hamburg.de</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="generic-base"/>
```
Comment (rmzelle) [inline journal-of-mountain-science.csl]: @nilsnerd, can you add:

```
<issn>1672-6316</issn>
<eissn>1993-0321</eissn>
```

here? (ISSNs obtained from http://www.springer.com/earth+sciences+and+geography/journal/11629)

Comment (rmzelle) [inline journal-of-mountain-science.csl]: @nilsnerd, can you also select a more appropriate category field to replace "generic-base"? Maybe `<category field="geography"/>`?

http://docs.citationstyles.org/en/stable/specification.html#appendix-i-categories

**Comment by rmzelle** [comment]
Closing because of inactivity. @nilsnerd, you can still make the requested changes; if you do that and leave a comment, we'll take another look.

**Pull request closed without merging**